### ISSUE TITLE
End location of `if` without semicolon

### ISSUE THREAD

**Issue by czosel** [glayzzle/php-parser#168]
I was surprised that (mind the empty line 2)
```php
<?php if ($foo) ?>

```
is parsed with `loc.end.line == 2`,
```json
...
    {
      "kind": "if",
      "loc": {
        "source": "if ($foo) ?>\n",
        "start": {
          "line": 1,
          "column": 6,
          "offset": 6
        },
        "end": {
          "line": 2,
          "column": 0,
          "offset": 19
        }
      },
      ...
```
While
```php
<?php if ($foo); ?>

```
is parsed with `loc.end.line == 1`:
```json
...
    {
      "kind": "if",
      "loc": {
        "source": "if ($foo);",
        "start": {
          "line": 1,
          "column": 6,
          "offset": 6
        },
        "end": {
          "line": 1,
          "column": 16,
          "offset": 16
        }
      },
      ...
```
Is this a bug, or actually intended? Thanks in advance!

**Comment by ichiriac** [comment]
I need to investigate more deeper but seems to be a bug. I'll start my vacations in 2 weeks, I'll work on a serious release then.

**Comment by czosel** [comment]
Perfect, thanks for looking into this! :+1:

**Comment by ichiriac** [comment]
that comes from the fact `?>` eats the next line return and acts exactly as a `;` so it was included into the `if` node - I'll fix the problem by ignoring this token - and handle it with the future `Script` node

**Comment by czosel** [comment]
Awesome, thanks! :+1:

**Comment by ichiriac** [comment]
This fix is now relased under 3.0.0-alpha.3
